locking/locks.go

```diff
@@ -154,6 +157,17 @@ func (c *Client) UnlockFileById(id string, force bool) error {
 	if err := c.cache.RemoveById(id); err != nil {
 		return fmt.Errorf("Error caching unlock information: %v", err)
 	}
+	path := unlockRes.Lock.Path
+	gitRoot, err := git.RootDir()
+	if err != nil {
+		return err
+	}
+	absPath := filepath.Join(gitRoot, path)
```

Comment:
> I'll review this because my fork may not be adhering to that aspect of the API.

Sounds good. I think my preference would be to back out of this change as well, but it may be good to keep anyway, so that LFS can better handle servers that implement the locking API incorrectly.

What are your thoughts?